cypress/integration/ui/item_context_menu/item_context_menu.js

```diff
@@ -0,0 +1,41 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps'
+import { dashboardItem as itemEl } from '../../../elements/dashboardItem'
```

Comment:
Why rename to `itemEl`?

Reply:
Name-wise to me I get the impression that this is the same element that keeps being reused (i.e. as if it was an instance of a class), but it seems like this represents all dashboard items and the item itself is targeted by providing an id. I think these two concepts are clashing a bit from a readability perspective.
I suggest either naming it something more generic, to not get the impression of being a specific unique element.
Or having it receive the itemUid as a prop, like `itemEl(chartItemUid).action.clickMenuButton()`

Reply:
In the second alternative I'd probably instantiate `dashboardItem` twice instead of passing the id all the time, like `const chartItemEl = dashboardItem (dashboards.Delivery.items.chart.uid)` and then use it like `chartItemEl.action...`